src/herbivoresAndCarnivores.js

```diff
@@ -1,19 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(herb) {
+    if (!herb.hidden && herb instanceof Herbivore) {
```

Comment:
your parameter name is `herb`, short for herbivore(?). but then you are checking whether it is a herbivore. seems strange